boards/components/tock_rubble/src/refcell_packet_queue.rs

```diff
@@ -0,0 +1,119 @@
+//! Safe, non-interrupt-driven packet queue written using RefCells.
+//!
+//! Copied from https://github.com/jonas-schievink/rubble/pull/137/files#diff-80283f6441097bbff4aad2c54957a72a.
```

Comment:
This is 0-clause BSD code and not Apache/MIT like the rest of Tock. Legally I think that's ok, but it should be documented here.